resources/views/components/interpretation.blade.php

```diff
@@ -1,20 +1,23 @@
-<div id="{{ $id }}" {{ $attributes->class(['interpretation', 'expander', 'stack']) }} x-data="{ expanded: false }">
-    @if ($videoSrc)
-        <button class="borderless" type="button" x-bind:aria-expanded="expanded.toString()"
-            x-on:click="expanded = !expanded">
-            @svg('tae-sign-language', 'icon--xl')
-            {{ __('Sign Language video') }}
-        </button>
-        <div class="stack interpretation__video" x-data="vimeoPlayer({
-            url: '{{ $videoSrc }}',
-            byline: false,
-            pip: true,
-            portrait: false,
-            responsive: true,
-            speed: true,
-            title: false
-        })" x-init="$watch('expanded', value => togglePlayback(value))"
-            @ended="player().setCurrentTime(0)" x-show="expanded" x-cloak>
-        </div>
-    @endif
-</div>
+@if (is_signed_language(locale()))
+    <div id="{{ $id }}" {{ $attributes->class(['interpretation', 'expander', 'stack']) }}
+        x-data="{ expanded: false }">
+        @if ($videoSrc)
```

Comment:
Should probably move the `$videoSrc` check into the first conditional, that way the parent `<div>` isn't rendered when there is no video provided but the sign language locale is set.

Reply:
Although doing so would mean that the links from the admin panel won't work if the sign language locale isn't set. I suppose we could force those URLs into one of the appropriate sign language locale. Not making the above change will cause a bit of a shift in the layout for the empty div.

Reply:
Resolved in https://github.com/accessibility-exchange/platform/pull/1329/commits/c6b0d60c1046d3e4c2eca51b629ffcdcf20080a1 and ffa73f86.